torchreid/models/squeezenet.py

```diff
@@ -6,16 +6,38 @@
 import torch.nn as nn
```

Comment:
@sholevs66  revert this file - you can checkout a specific file from specific commit hash - googleit